rules/Php80/Rector/Class_/AnnotationToAttributeRector.php

```diff
@@ -369,4 +379,28 @@ private function collectDoctrineTagAndAnnotationToAttributes(
 
         return $doctrineTagAndAnnotationToAttributes;
     }
+
+    private function changeResolvedClassInContext(
+        AnnotationToAttribute $annotationToAttribute,
+        DoctrineAnnotationTagValueNode $docNode
+    ): DoctrineAnnotationTagValueNode {
+        foreach (self::CHANGE_RESOLVED_CLASS_IN_CONTEXT as $item) {
+            $docNodeValue = $docNode->getValue($item['value']);
+            if ($annotationToAttribute->getTag() !== $item['tag'] || ! $docNodeValue) {
+                continue;
+            }
+
+            if (! ($docNodeValue instanceof \Rector\BetterPhpDocParser\ValueObject\PhpDoc\DoctrineAnnotation\CurlyListNode)) {
+                continue;
+            }
+            $toTraverse = [$docNodeValue->getValues()[0], $docNodeValue->getOriginalValues()[0]];
```

Comment:
`isset()` for `[0]` index is needed to avoid the index removed by other rule or not exists as empty on a race condition.